upm-tools/AssemblyDefinitions/BugsnagEditor.asmdef

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
if you didnt already, please follow the steps in the CONTRIBUTING.md to build a local UPM package and check that this change doesnt break the package